compiler/src/dmd/root/port.d

```diff
@@ -85,13 +99,14 @@ extern (C++) struct Port
             int res = _atoflt(&r, s);
             if (res == _UNDERFLOW || res == _OVERFLOW)
                 errno = ERANGE;
+            version (CRuntime_DigitalMars) __locale_decpoint = save;
```

Comment:
`version (CRuntime_DigitalMars)` and `version (CRuntime_Microsoft)` are mutually exclusive.